nrf_security/configs/legacy_crypto_config.h.template

```diff
@@ -1906,16 +1630,6 @@
  */
 #cmakedefine MBEDTLS_SSL_SESSION_TICKETS
 
-/**
- * \def MBEDTLS_SSL_EXPORT_KEYS
- *
- * Enable support for exporting key block and master secret.
- * This is required for certain users of TLS, e.g. EAP-TLS.
- *
- * Comment this macro to disable support for key export
- */
-#define MBEDTLS_SSL_EXPORT_KEYS
-
```

Comment:
The feature is needed for OpenThread. The OpenThread based samples don't compile without it.

Reply:
Will fix with proper configuration that is exposed

Reply:
fixed